Classes/Setup.php

```diff
@@ -163,28 +163,38 @@ public static function execute()
      */
     public static function schema(array $sql) : array
     {
-        $tables = [];
-        $conn = self::context()->db();
+        $ctx = self::getContext();
+        $dbm = $ctx->getDatabaseManager();
+        $connectionNames = array_keys($ctx->getConfig()->get( 'resource'));
+        $connectionNames = array_filter($connectionNames, fn (string $key): bool => str_starts_with($key, 'db'));
 
-        foreach (['fe_users_', 'madmin_', 'mshop_'] as $prefix) {
-            $result = $conn->create('SHOW TABLES like \'' . $prefix . '%\'')->execute();
+        foreach ($connectionNames as $connectionName) {
+            $conn = $dbm->acquire($connectionName);
```

Comment:
It's just `$conn = $ctx->db($connectionName);` in the current version

Reply:
I also removed fetching the database manager and releasing the connection. It looked like the API to retrieve the manager no longer existed.

Reply:
Yes, API changed and is much simpler now :-)